xunit-autolabeler-v2/ast_parser/python/drift_data_dict.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def make_drift_data_dict(
```

Comment:
Consider using a NamedTuple for this, better performance, has type checking and can be accessed by attribute name.

class DriftData(typing.NamedTuple):
  name: str
  class_name: str
  ...
  flask_http_methods: List[str] = []


my_drift_datum = DriftData('A name', 'A class name')

Reply:
Also this could be declared in the only file is being used right now (unless you are planning to re-use it in multiple modules).

Reply:
@tmatsuo WDYT? I know you were against using an object here.

Reply:
I was against using an object just because you had some troubles making it work.
Using NamedTuple sounds great (TIL).

Reply:
Done. 🙂